x-pack/legacy/plugins/siem/server/lib/detection_engine/signals/get_filter.ts

```diff
@@ -39,7 +42,8 @@ export const getQueryFilter = (
     f => f && !esFilters.isFilterDisabled(f)
   );
 
-  return esQuery.buildEsQuery(indexPattern, queries, enabledFilters, config);
+  const a = esQuery.buildEsQuery(indexPattern, queries, enabledFilters, config);
+  return a;
```

Comment:
What's up with the `a` here? Is this left over from debugging something?

Reply:
Shoot - yes, that was leftover debugging. Thought I'd removed it when I did `git restore` on the file. Put back to what it was. Thanks for catching that.